### DISCUSSION TITLE
Image-Upload

### DISCUSSION THREAD

**Discussion by rueveydaakuel** [vercel/next.js#52439] [Help]
### Summary

Hello,
I urgently need help. I'm building a REACT-based app and recently integrated Cloudinary into my app to enable image uploads for users. After testing it on localhost:3000, everything works fine. However, when I try to deploy my app on Vercel or perform the image upload on Vercel, I receive an error message in the console, and the image upload doesn't occur.
![Bildschirmfoto 2023-07-08 um 12 50 29](https://github.com/vercel/next.js/assets/130995493/cf508308-36c2-4850-a609-151d9b214675)

**Comment by zerosym** [comment]
Your cloudinary API URL seems malformed, like you tried to interpolate an env var that is missing.
